Resources/views/full/file.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends "eZDemoBundle::pagelayout.html.twig" %}
+
+{% block content %}
+    {% set fileSize = content.getField( 'file' ).value.fileSize|ez_file_size( 4 ) %}
```

Comment:
You should use the ez_field_value Twig helper to get the current language.
Otherwise you get always the main language.